code/web/src/modules/user/ProfileForm.js

```diff
@@ -0,0 +1,237 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+// UI Imports
+import { Grid, GridCell } from '../../ui/grid'
+import Input from '../../ui/input/Input'
+import { H3, H4 } from '../../ui/typography'
+import Icon from '../../ui/icon'
+import Button from '../../ui/button'
+import { grey, grey2 } from '../../ui/common/colors'
+import { white } from "../../ui/common/colors"
+
+
+// App Imports
+import userRoutes from '../../setup/routes/user'
+import { updateUserInfo } from './api/actions'
+import { routeImage } from '../../setup/routes'
+import { renderIf } from '../../setup/helpers'
+import { upload, messageShow, messageHide } from '../common/api/actions'
+
+
+class ProfileForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userDetails: {
+        id: 0,
+        image: '',
```

Comment:
I spoke with Kara about this, but maybe in the future we will want to add the user's current image as well with `this.props.user.details.image`.

Reply:
That seems right to me as well @jdcarey128 I think we have to wait until we have the connection with BE and FE done!